### PR TITLE
Fix: Create random users in batches in create-data script

### PR DESCRIPTION
**Pull request by mmioana**
## Description

Refactored the create-data script in order to minimise failure while creating too many users all at once.
The random users creation is batched in chunks of 3.

Also, added a new method  `graphqlRequestPreconfigured` to enclose the `GRAPHQL_URI` and `API_KEY` used for the `graphqlRequest`

Add character limits for 
- user fields: displayName, bio and location
- colony fields: displayName and objective.title

## Testing

* Step 1. Run `npm ci`
* Step 2. Run `npm run dev`
* Step 3. Run `node ./scripts/create-data.js --yes`
* Step 4. Check if there are any errors while creating the data

Resolves #[2606](https://github.com/JoinColony/colonyCDapp/issues/2606)

### CODE REVIEW

**Review by bassgeta — APPROVED** [review]
image
Data gets populated for me, script runs fine, so nice job!
Just left a tiny nitpick for the objective description length, since we're already clipping all the texts :)

**Review by iamsamgibbs — APPROVED** [review]
Looks good to me! Script runs correctly and users are created.

Screenshot 2024-07-02 at 15 35 54
Screenshot 2024-07-02 at 15 36 10
Screenshot 2024-07-02 at 15 36 58

At some point we might want to update the example text in `tempColonyData.js` to fit the actual character limits but that is very much a nice to have.

File: scripts/create-data.js [hunk 1]
```diff
@@ -195,6 +207,9 @@ const getColonyMetadata = /* GraphQL */ `
   }
 `;
 
+const graphqlRequestPreconfigured = async (queryOrMutation, variables) => 
+  graphqlRequest(queryOrMutation, variables, GRAPHQL_URI, API_KEY)
```
Comment (davecreaser) [inline scripts/create-data.js:211]: Nice! Big fan of this. 📈

**Review by davecreaser — APPROVED** [review]
Good stuff, the script is still working as expected and creating all the users. Approved!
Screenshot 2024-07-02 at 16 52 15
Screenshot 2024-07-02 at 16 58 41

**Review by bassgeta — APPROVED** [review]
image
Still works, planex status = populated.
Nice work on this, let's get this one merged!

**Review by iamsamgibbs — APPROVED** [review]

**Review by davecreaser — APPROVED** [review]

**Pull request merged**